clients/ts/signalr/src/DefaultHttpClient.ts

```diff
@@ -45,4 +45,12 @@ export class DefaultHttpClient extends HttpClient {
 
         return this.httpClient.send(request);
     }
+
+    // @ts-ignore
+    private getCookies(url: string): string {
```

Comment:
Have to do it this way so we don't make a breaking change by requiring this function on the base class `HttpClient`

Reply:
Couldn't you define it as non-abstract and have it return an empty list?

Reply:
[You can override methods just fine in subclasses...](https://www.typescriptlang.org/play/index.html#src=abstract%20class%20Base%20%7B%0D%0A%20%20%20%20method2()%3A%20string%20%7B%20return%20%22Base%22%20%7D%0D%0A%7D%0D%0A%0D%0Aclass%20Derived%20extends%20Base%20%7B%0D%0A%20%20%20%20method2()%3A%20string%20%7B%0D%0A%20%20%20%20%20%20%20%20return%20%22Derived%22%0D%0A%20%20%20%20%7D%0D%0A%7D%0D%0A%0D%0Aclass%20NonOverride%20extends%20Base%20%7B%0D%0A%7D%0D%0A%0D%0Aconsole.log(new%20NonOverride().method2())%3B%0D%0Aconsole.log(new%20Derived().method2())%3B) In TypeScript, everything is `virtual` ;)